Gemfile.lock

```diff
@@ -31,8 +31,8 @@ PATH
   specs:
     actioncable (5.0.0.beta1)
       actionpack (= 5.0.0.beta1)
-      celluloid (~> 0.17.2)
       coffee-rails (~> 4.1.0)
+      concurrent-ruby (~> 1.0.0)
```

Comment:
@arthurnn  `sucker_punch` is moving from Celluloid to `concurrent-ruby`. The in-progress work is [here](https://github.com/brandonhilkert/sucker_punch/pull/145). I think @brandonhilkert will be releasing a beta fairly soon.